packages/manager/src/components/Snackbar/Snackbar.tsx

```diff
@@ -29,7 +29,7 @@ const useStyles = makeStyles((theme: Theme) => ({
     borderLeft: `6px solid ${theme.palette.primary.main}`,
   },
   success: {
-    borderLeft: `6px solid ${theme.palette.primary.main}`,
+    borderLeft: `6px solid ${theme.color.green}`,
```

Comment:
Instead of using `theme.color.green` can we fix `theme.palette.success.*` to actually be green and use it here?

```suggestion
    borderLeft: `6px solid ${theme.palette.success.main}`,
```